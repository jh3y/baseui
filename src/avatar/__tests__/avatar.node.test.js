/*
Copyright (c) 2018-2019 Uber Technologies, Inc.

This source code is licensed under the MIT license found in the
LICENSE file in the root directory of this source tree.
*/
// @flow

import * as React from 'react';
import {render} from 'enzyme';

import {Avatar} from '../index.js';

describe('Avatar', () => {
  it('applies expected accessibility attributes to img element', () => {
    const name = 'user name';
    const image = render(<Avatar name={name} src="valid-img-src.png" />);
    expect(image).toMatchSnapshot('SSR Avatar');
  });
});
