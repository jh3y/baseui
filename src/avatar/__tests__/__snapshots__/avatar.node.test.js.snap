// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar applies expected accessibility attributes to img element: SSR Avatar 1`] = `
<div
  data-baseweb="avatar"
  styled-component="true"
  test-style="[object Object]"
>
  <img
    alt="user name"
    src="valid-img-src.png"
    styled-component="true"
    test-style="[object Object]"
  />
</div>
`;
